clrenv/lazy_env.py

```diff
@@ -3,16 +3,35 @@
 import os.path
 from os import environ
 from glob import glob
-from itertools import chain, groupby
+from itertools import chain
 import shlex
 import sys
+import logging
 
 from munch import Munch, munchify
 import yaml
 
 from .path import find_environment_path, find_user_environment_paths
 from functools import reduce
 
+logger = logging.getLogger(__name__)
+
+# Flag to prevent clrenv from throwing errors
+#  if it cannot connect to the Parameter Store API.
+OFFLINE_FLAG = 'CLRENV_OFFLINE_DEV'
+
+class LazyMunch(Munch):
```

Comment:
With the offline flag, I'm not sure whether I need to do this lazy-loading anymore. Leaning towards removing.